store/engine/raft/node.go

```diff
@@ -90,7 +90,6 @@ type Node struct {
 var _ engine.Engine = (*Node)(nil)
 
 func New(config *Config) (*Node, error) {
-	config.init()
```

Comment:
Why?

Reply:
the initial value of `Raft.config` is in the `config.yaml`, so we don't call `config.init()` again, this can keeps it consistent with other storage_type (etcd & zk).

Reply:
@fukua95 Technically, a user can use a different config file than the one we have in the repository (this one is just an example or so). The of the `config.init` function (if I understood it right) is to check the loaded config and set defaults if needed.

Reply:
I recover it.